README.md

```diff
@@ -25,12 +25,16 @@ In this challenge, create a web page that presents a styled list of Star Wars ch
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
+    React is a user interface library that helps to quickly and efficiently handle and display large amounts of information. It allows companies like facebook and AirBNB to take their current data and update it without needing to reload the whole page. They can update just what needs to be updated.
 
 - [ ] What does it mean to _think_ in react?
+    React has pattern to its process that requires you to think differently then when other libraries.
 
 - [ ] Describe state.
+    State is just the data we need for the website. It's called state because we can update the data to a new state of data.
```

Comment:
Here's more information about *Thinking* in React.
https://reactjs.org/docs/thinking-in-react.html